### ISSUE TITLE
[Bug]: Update to Compound Brand Stroke values

### ISSUE THREAD

**Issue by redongreen** [microsoft/fluentui#23837]
### Library

React Components / v9 (@fluentui/react-components)

### System Info

```shell
https://master--628d031b55e942004ac95df1.chromatic.com/?path=/docs/theme-theme--border-radii#colors
```

### Reproduction

https://master--628d031b55e942004ac95df1.chromatic.com/?path=/docs/theme-theme--border-radii#colors

## Context
With the Brand Communication Blue palette update (April 22), we updated both `colorCompoundBrandBackground` and  `colorCompoundBrandForegroundground` colors to use the intended dark theme colors. 

However, we recently noticed that `colorCompoundBrandStroke` was not updated to use the new color assignment for dark theme.

Please update `colorCompoundBrandStroke` and its interactive states to use the intended brand communication blue colors.

Components impacted
- Checkbox
- RadioGroup
- Input
- Tablist
- TextArea

## Actual Behavior

**Dark theme** 
Compound Brand Stroke = Comm 90 = #2886DE
Compound Brand Stroke Hover = Comm 100 = #479EF5
Compound Brand Stroke Pressed = Comm 80 = #0F6CBD

<img width="969" alt="image" src="https://user-images.githubusercontent.com/20573265/177884654-8864d948-dfe3-4d34-876d-b3c0764d9b1e.png">

## Expected Behavior

**Dark mode** 
Compound Brand Stroke = Comm 100 = #479EF5
Compound Brand Stroke Hover = Comm 110 = #62ABF5
Compound Brand Stroke Pressed = Comm 90 = #2886DE

### Requested priority

High

### Are you willing to submit a PR to fix?

no

### Validations

- [X] Check that there isn't already an issue that reports the same bug to avoid creating a duplicate.
- [X] The provided reproduction is a minimal reproducible example of the bug.

**Comment by tudorpopams** [comment]
Hello @microsoft/cxe-red , can we follow up with design about how important this change is and whether or not this will be destabilising for partners.

**Comment by spmonahan** [comment]
Looks like @miroslavstastny fixed this in #24608
